include/messages/MessageHelper.h

```diff
@@ -8,6 +8,7 @@ namespace Messages::MessageHelper
 {
     void placeMessageSizeInByteBuffer(Common::Types::ByteVector& byte_buffer, std::size_t size_bytes);
     void placeMessageTypeInByteBuffer(Common::Types::ByteVector& byte_buffer, MessageType message_type);
+    void place32BitValueInByteBuffer(Common::Types::ByteVector& byte_buffer, std::size_t offset, std::uint32_t value);
```

Comment:
Why 32bit value here? It should be arbitrary number of bytes since this is the message payload?

Reply:
The problem comes when we have types that are intrinsically multiple bytes. An int with 4 bytes needs to care about endianness where as an array of 4 bytes (like an array of chars) does not need to care about endianness. I'm open to suggestions if you can think of a way around this.

Reply:
The other way would be to have this function accept a list of bytes to place into a buffer. But that means we need to break all mulit byte types into a list of bytes before passing it to this function.

- short (usually 2 bytes)
- int (usually 4 bytes) 
- double (usually 8 bytes)